apps/portal-dashboard/tailwind.config.ts

```diff
@@ -68,6 +68,9 @@ const config = {
           foreground: "hsl(var(--card-foreground))"
         }
       },
+      backgroundImage: {
+        "hero-pattern": "url('./images/lume-bg.png')"
```

Comment:
Would be great to have this as a theme too! Maybe coming fron a css var, we have quite a lot of different images for the different themes, if we treat them as slots we could also have them dynamically loading

Reply:
Naming according to page is good!